admin-dev/themes/new-theme/js/pages/webservice/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import Grid from "../../components/grid/grid";
+import BulkActionCheckboxExtension from '../../components/grid/extension/bulk-action-checkbox-extension';
+import SubmitBulkExtension from '../../components/grid/extension/submit-bulk-action-extension';
+import FiltersResetExtension from '../../components/grid/extension/filters-reset-extension';
+
+const $ = window.$;
+
+$(() => {
+  const webserviceGrid = new Grid('webservice');
+
+    webserviceGrid.addExtension(new BulkActionCheckboxExtension());
```

Comment:
Fix indent :)

Reply:
time to add a linter/fixer for js files isn't it?

Reply:
https://github.com/PrestaShop/PrestaShop/pull/9385 :roll_eyes:

Reply:
it's up to @eternoendless, I hate javascript no matter how it is formatted :trollface:

Reply:
Es6/Es7 ftw dude!

Reply:
:vomit: javascript